persistence/src/main/java/com/epam/esm/dao/mapper/TagMapper.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.esm.dao.mapper;
+
+import com.epam.esm.entity.Tag;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class TagMapper implements RowMapper<Tag> {
+    @Override
+    public Tag mapRow(ResultSet rs, int rowNum) throws SQLException {
+        return Tag.builder()
+                .id(rs.getLong("id"))
+                .name(rs.getString("name"))
```

Comment:
Id is a unique field by which one record is returned. What is the additional check by name for?